stylesheets/partials/_index-applicant.scss

```diff
@@ -1,5 +1,6 @@
 @forward "colors";
-@forward "base";
+@forward "base-applicant";
 @forward "buttons";
 @forward "alerts";
 @forward "photo-grid";
+@forward "common";
```

Comment:
suggestion (take it or leave it): If `common` was listed first, it would allow anything else to override it, or more likely to use any variables or anything defined in `common`. (But none of that is likely to really apply here, practically…)

Reply:
Great point. It also makes the index files more consistent.